backend/src/controllers/review.controller.ts

```diff
@@ -69,15 +69,15 @@ export async function deleteReviewHandler(req, res) {
       },
     });
 
-    if (movieDelete.userId == req.user.id) {
-      res.status(401).json;
+    if (movieDelete.userId != req.user.id) {
+      res.status(401).json();
```

Comment:
this json should return a message about how you are trying to delete a review that you dont own